src/app/components/AddDocumentForm/AddDocumentForm.module.css

```diff
@@ -0,0 +1,36 @@
+.saveContainer {
+  display: grid;
+}
+
+.filenameInput {
+  background: var(--color-tertiary);
+  border-radius: 10px;
+  color: var(--color-text-secondary);
+  text-align: center;
+  padding: 0.5rem;
+  width: 200px;
+  border: solid var(--color-text-secondary) 2px;
+  margin: 1rem;
+}
+
+.saveButton {
+  background: var(--color-secondary);
+  border-radius: 10px;
+  color: var(--color-text-primary);
+  font-size: 1.25rem;
+  text-align: center;
+  font-weight: 800;
+  padding: 0.5rem;
+  width: 200px;
```

Comment:
A fixed width for a button is not a good idea. If you increase the root font-size or change the text, you will face an overflow.